karma.conf.js

```diff
@@ -44,6 +44,14 @@ module.exports = function(config) {
         },
         webpack: {
             mode: "none",
+            plugins: [
```

Comment:
is the config any difference with `./webpack.config.js`? if no, we can just import it here:

```js
webpack: require("./webpack.config.js"),
```

Reply:
The webpack configuration in root is for building ESLint as a library, while in Karma, it's for browser testing as an application.